eve/workers/pod-linter/Dockerfile

```diff
@@ -1,19 +1,21 @@
 # CI lint container - Fedora for recent ShellCheck version
-FROM fedora:29
+FROM fedora:31
```

Comment:
this bump (and the install of `git` and `findutils`) is related to the bump of the operator SDK?

If yes, would be nice to have a little explanation on why those new reqs in the commit message.
If no, could be done in a dedicated commit.